packages/ember-metal/lib/tracked.ts

```diff
@@ -208,8 +199,8 @@ function descriptorForDataProperty(key, descriptor) {
       return this[shadowKey];
     },
 
-    set(newValue) {
-      tagFor(this).inner.dirty();
+    set(newValue: any) {
+      tagFor(this).inner!['dirty']();
```

Comment:
This specifically "feels bad", but I'm not sure what the right thing to do here is.  The changes here squelch the type errors (which fail CI), but don't _really_ ensure that this works properly.

At the moment, this file is complete unused (the actual implementation lives in `./tracked.js`), but I'd love to followup with a more correct fix for this..